packages/lib/src/vis/shared/AxisSystemProvider.tsx

```diff
@@ -60,14 +60,13 @@ function AxisSystemProvider(props: PropsWithChildren<Props>) {
       .setPosition(width / 2, height / 2); // account for shift of (0,0) position (center for camera, top-left for HTML)
   }, [availableSize]);
 
-  const camera = useThree((state) => state.camera);
-  const worldToHtml = useCallback(
```

Comment:
Converting from world space to camera space with `someVec3.project(camera)`, and back with `unproject`, must now be done inside grabbers. We could imagine providing two grabbers called `grabCameraPtFromWorld` and `grabWorldPointFromCamera` or something like that, but I don't see a use case for them at the moment. I feel like camera space is never a source/target space; only an in-between space. That's why I went with the more specific `grabHtmlPoint` instead, for data-to-HTML conversions (and we could later add `grabHtmlPointFromWorld` for world-to-HTML conversions).